applications/rpg_table_helper/pubspec.yaml

```
@@ -57,3 +63,9 @@ fonts:
 
 flutter_intl:
   enabled: true
+
+flutter_launcher_icons:
+  android: "launcher_icon"
+  ios: true
+  image_path: "assets/icons/icon.jpg"
+  min_sdk_android: 21 # android min sdk min:16, default 21
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)